fs/metrics/manager/manager.go

```diff
@@ -0,0 +1,44 @@
+/*
+   Copyright The Soci Snapshotter Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package manager
```

Comment:
I'm not sure about this package name. `manager` could be a valid package for just about anything.


Is there a strong reason for not keeping this in the top level `metrics` package? `metrics.FuseObservabilityManager` seems pretty reasonable.

ref: https://go.dev/blog/package-names

Reply:
Yeah, this was sort of a placeholder name, mainly because I couldn't think of a more concise/fitting name. I was thinking of this initial `FuseObservabilityManager` construct to be a stepping stone, where eventually we would end up with a well defined interface for `manager`. Since it isn't well defined here I can move it up to `metrics`.